src/staging/src/styles/megaMenu/megaMenu.scss

```diff
@@ -0,0 +1,561 @@
+@import '../variables.scss';
+
+/* ---------------- HEADER ----------------- */
+
+
+/* TODO Move to global.scss? */
+
+.bc-parks-logo {
+  max-height: 50px;
+  max-width: 199px;
+}
+.bc-parks-wordmark {
+  height: 50px;
+  max-width: 199px;
+}
+a.btn.book-campsite-btn {
+  background-color: $colorWhite;
+  border: solid 1px transparent;
+  color: $colorBluePrimary;
+  font-size: 13px;
+  padding: 0px 30px;
+  line-height:40px;
+  height:40px;
+  border-radius:20px;
+  &:hover {
+    background-color: $colorWhite;
+    color: $colorBluePrimary;
+    text-decoration: underline;
+    outline: 1px solid $colorGoldPrimary;
+  }
+  @media screen and (max-width: $desktopBreakpoint) {
+    margin-right:50px;
+    padding: 0px 10px;
+    line-height:30px;
+    height:30px;
+    border-radius:15px;
+  }
+
+}
+
+@media screen and (min-width: $desktopBreakpoint) {
+
+    .navbar--header {
+        max-width: 1140px; 
+        margin: 0px auto;
+        padding: 0px 20px;
+        height: 100px;
+    }
+}
+
+/* TODO scope these properly: */
+
+nav {
+   ul, li {
+     list-style: none;
+     padding: 0;
+     margin: 0;
+  } 
+  a {
+    display: block;
+    text-decoration: none;
+  }
+}
+
+
+
+/* -------------------- START MOBILE STYLING ----------------- */
+@media screen and (max-width: $desktopBreakpoint) {
+    .menu-toggle {
+        position:absolute;
+        top:52px;
+        right:20px;
+        color:$colorWhite;
+        font-size: 20px;
+
+        .menu-open {
+            display: block;
+        }
+        .menu-close {
+            display:none;
+        }
+
+        &.menu-toggle--open {
+            .menu-open {
+                display: none;
+            }
+            .menu-close {
+                display:block;
+            }
+        }
+    }
+
+    .mega-menu.menu-mode--responsive.menu-open{
+        height:1000px;
```

Comment:
There seems to be some unnecessary empty space below the last item of the megamenu on mobile view

Reply:
yes, added this in discussion with harry. before the menu would end just below the last item and look weird, this way the white bg will fill a phone (but not necessarily over all the content, depending on length). will resolve this, however re-open if you think it needs more discussion